examples/kubernetes/README.md

```diff
@@ -3,4 +3,7 @@
 If you're using this with the openshift image in test/images then you'll
 need to run the following to access services:
 
-$ sudo route add -net 172.30.0.0/16 gw 10.111.112.101
+    $ sudo ip route add 172.30.0.0/16 via 10.111.112.101
+
+In order for image download to work you also need to drop the `*.docker.io`
```

Comment:
I'd suggest a better explanation here. Maybe something like:

This vm is preloaded with the docker image needed to run the cockpit integration tests. In order to avoid test flakes due to network issues, any further image downloads are blocked by redirecting common docker registry domains to localhost in /etc/hosts. If you need to download new images while manually testing you will need to remove those entries from /etc/hosts.